pkg/middleware/jwt_session.go

```diff
@@ -75,24 +65,26 @@ func (j *jwtSessionLoader) getJwtSession(req *http.Request) (*sessionsapi.Sessio
 		return nil, nil
 	}
 
-	rawBearerToken, err := j.findBearerTokenFromHeader(auth)
+	token, err := j.findTokenFromHeader(auth)
 	if err != nil {
 		return nil, err
 	}
 
+	errs := []error{fmt.Errorf("unable to verify jwt token: %q", req.Header.Get("Authorization"))}
```

Comment:
I think this could do with a comment that explains it is only used if no session is loaded, alternatively you could initialise and empty list here, then later

```
// If we got here, no session loader has been able to find a session
errs = append([]error{fmt.Errorf("unable to verify jwt token: %q", req.Header.Get("Authorization")}, errs...)
return nil, errors.NewAggregate(errs)
```

On a second note, we should probably implement something that redacts strings when we are logging them, printing peoples tokens into logs probably isn't very good from a security perspective

Reply:
Very good point - I just brought over the old log message that had the `token` from before and appended OIDC verifier chain errors.

The initial message should definitely scrub or remove that token.

I'll completely remove for now, if it comes up again if people want it for debuging, we can do some logic where we strip the signature maybe? Feels hacky...

Reply:
All set: f55d27e